src/is_wire/core/message.py

```diff
@@ -335,27 +343,33 @@ def pack(self, obj):
                 "Serialization to '{}' type not implemented".format(
                     self.content_type.name))
 
-    def unpack(self, schema):
+    def unpack(self, schema=dict):
         """ Deserializes the content of the message using the given schema.
         If the message has no content_type, the protobuf format is used.
         Args:
             schema (type): type of the protobuf object to be deserialized.
         Returns:
             schema: deserialized instance of the object of type schema.
         """
+
+        isDict = schema is dict
+        if isDict:
+            schema = Struct
+
         obj = schema()
         if not self.has_content_type():
             self.content_type = ContentType.PROTOBUF
 
         if self.content_type == ContentType.PROTOBUF:
             obj.ParseFromString(self.body)
-
         elif self.content_type == ContentType.JSON:
-            obj = json_format.Parse(self.body, obj)
-
+            obj = pb.Parse(self.body, obj)
         else:
             raise NotImplementedError(
                 "Deserialization from '{}' type not implemented".format(
                     self.content_type.name))
 
+        if isDict:
```

Comment:
I had the following result when pack a protobuf message and afterwards unpack without specify a schema.

```python
>>> from is_msgs.common_pb2 import Pose, Position
>>> from is_wire.core import Message
>>> p = Pose(position=Position(x=1))
>>> msg = Message()
>>> msg.pack(p)
>>> msg.unpack()
{'': None}
>>> msg.unpack(Pose)
position {
  x: 1.0
}
```
I think that doesn't make sense, when receiving a message with protobuf content type, try to convert to dict if no protobuf schema given. Not sure what would be the correct the expected behavior. I can only think on these possibilities:
- Let the user figure out with the returned dict, that a schema is needed to unpack this message;
- Raise an exception with that information;